beacon-chain/sync/initial-sync/initial_sync_test.go

```diff
@@ -276,9 +276,7 @@ func connectPeerHavingBlocks(
 	p := p2pt.NewTestP2P(t)
 
 	p.SetStreamHandler("/eth2/beacon_chain/req/beacon_blocks_by_range/1/ssz_snappy", func(stream network.Stream) {
-		defer func() {
-			assert.NoError(t, stream.Close())
-		}()
+		defer stream.Close()
```

Comment:
This is a symptom of async closing. Specifically, some of the tests disconnect from this peer immediately after making the request and before the stream closure can be flushed. This doesn't really hurt anything in practice.

Reply:
Ah, fair enough. I can drop the error.